sdk/spring/spring-cloud-azure-actuator/src/main/java/com/azure/spring/cloud/actuator/cosmos/CosmosHealthIndicator.java

```diff
@@ -45,18 +47,28 @@ public CosmosHealthIndicator(CosmosAsyncClient cosmosAsyncClient, String databas
 
     @Override
     protected void doHealthCheck(Builder builder) {
-        CosmosDatabaseResponse response = this.cosmosAsyncClient.getDatabase(database)
-            .read()
-            .block(timeout);
-
-        if (response != null) {
-            LOGGER.info("The health indicator cost {} RUs, cosmos uri: {}, dbName: {}",
-                response.getRequestCharge(), endpoint, database);
-        }
-        if (response == null) {
-            builder.down();
+        if (database == null) {
+            builder.status(Status.UNKNOWN).withDetail("Database not configured",
+                "The option of `spring.cloud.azure.cosmos.database` is not configured!");
         } else {
```

Comment:
I would prefer to return in this if

Reply:
add a `return` in this block will bring a warn for this method return is void.  Means this is unnessary.

Reply:
How about return builder.build()?